interfaces/standard/matlab_interface.py

```diff
@@ -29,70 +30,115 @@ def __init__(self):
             self.cls_str = 'cls'
         else:
             self.cls_str = 'clear'
+
         self.clear()
+
         if not import_fail:
-            print("Starting Matlab...")
-            self.eng = matlab.engine.start_matlab()
+            try:
+                self.eng = matlab.engine.start_matlab()
+                intro = '''\
+                MATLAB Interactive Terminal (R{release})
+                
+                To get started, type one of these commands:
+                    helpwin          Provide access to help comments for all functions
+                    helpdesk         Open help browser
+                    demo             Access product examples in Help browser
+                
+                For product information, visit https://www.mathworks.com.\
+                '''.format(release=self.release())
+                print(dedent(intro))
+
+            except Exception as e:
+                self.clear()
+                print("MATLAB Engine for Python exited prematurely.")
+                print(str(e))
+
         else:
-            print("Could not start Matlab")
+            print("Launching MATLAB failed: Error starting MATLAB process in MATLAB Engine for Python.")
 
     def clear(self):
         os.system(self.cls_str)
 
+    def release(self):
+        release_str = "version '-release';"
```

Comment:
```suggestion
        release_str = "version('-release');"
```
This instruction fails. When trying directly on Matlab, I see that it fails when trying to assign it to a variable (ie `a = version '-release';"). The statement alone works, and assigns the value to ans correctly. Weird. But using `version('-release');' seems to work :)